src/components/policy/policy_external/include/policy/policy_types.h

```diff
@@ -168,13 +168,13 @@ struct CheckPermissionResult {
   */
 struct EndpointData {
   explicit EndpointData(const std::string& url_string = "")
-      : app_id("default") {
+      : app_policy_id("default") {
     if (false == url_string.empty()) {
       url.push_back(url_string);
     }
   }
   std::vector<std::string> url;
-  std::string app_id;
+  std::string app_policy_id;
```

Comment:
@ZhdanovP in external policy used the same duplicate of EndpointData
I suppose you should rename app_id --> app_policy_id there too.

Reply:
@EKuliiev I have already done it.

Reply:
@ZhdanovP yep. Now I see. Sorry